### ISSUE TITLE
BUG: A11Y labels still appear on TH when class "no-sort" is set

### ISSUE THREAD

**Issue by tmorehouse** [tofsjonas/sortable#88]
when a `<TH>` has class `no-sort`, and the a11y addon is used, the table header cell still has A11Y attributes applied. Specifically `tab-index="0"` and `aria-label="..."`.

Expected behaviour:
When class `no-sort` is applied to the `<TH>` element, the `tab-index` would be removed or set to `-1`, and the `aria-label` will be removed or set to `""` (or have a string such as `colum foo cannot be sorted`).
